lib/Conversion/TritonGPUToLLVM/ReduceOpToLLVM.cpp

```diff
@@ -431,10 +431,12 @@ struct ReduceOpConversion
         unsigned shuffleIdx = N;
 #ifdef USE_ROCM
         if (inMfma && inMfma.getIsTransposed()) {
```

Comment:
@oplavsic 
Could you check if I understood this part correctly.

FA tutorial test passes, but still